packages/katello/katello/katello.spec

```
@@ -136,6 +133,9 @@ Provides a federation of katello services
 # the files section is empty, but without it no RPM will be generated
 
 %changelog
+* Thu Apr 08 2021 Eric D. Helms <ericdhelms@gmail.com> - 4.1.0-0.5.master
+- Drop requirement on hammer-cli-katello
```

Shouldn't this be broader?
```suggestion
- Drop requirement on Hammer CLI packages
```

---

Could be, the other hammer packages are redundantly declared given hammer-cli-katello already requires them.

---

Sure, but as a user who reads the changelog I think this would be easier to understand. Before I got Hammer, now I don't.